README.md

```diff
@@ -1,38 +1,67 @@
 # Thali Cordova Plugin
 
 <Work on progress, not near release quality !!>
+
 ## Goals
-The Thali Cordova Plugin is an easy-to-use Cordova plugin for building peer-to-peer (P2P) networking
-apps on Android and iOS. 
 
-The Thali Cordova Plugin is layered on the JXcore Cordova plugin, which uses JXcore to allow one to build
-mobile applicatons in JavaScript for Node.JS.  
+The Thali Cordova Plugin is an easy-to-use [Cordova](http://cordova.apache.org/) plugin for building peer-to-peer (P2P) networking apps on Android and iOS.
```

Comment:
I'm not going to argue this because once again I didn't write this originally, but will keep some changes in mind here.